internal/provider/provider.go

```diff
@@ -67,14 +67,14 @@ func (p *provider) Configure(
 		return
 	}
 
-	var apiKey string
+	var apiKeyOrJWT string
```

Comment:
Should we just use a different env var for the jwt? What do we gain from reusing the api key var for both?

Reply:
For the end-user, they need not deal with multiple envs.

Reply:
I'm not really convinced. 

I guess it's not a big deal either way. 

Usually I try to name things for what they are. It may feel strange for a customer to pass a jwt token into an env var called COCKROACH_API_KEY.  They might think, is this right? And need to double check the docs. But if the env var was COCKROACH_AUTH_JWT or something like this, it feels more natural. I won't block the review on it but it would be good for @marksoper to weigh in from a product perspective.

Reply:
Let's use a separate variable instead of overloading the api key. The benefit of differentiating between the two different concept outweighs the convenience of having a single variable serve both purposes.

Reply:
Done